src/ir/module-splitting.cpp

```diff
@@ -409,6 +434,21 @@ void ModuleSplitter::thunkExportedSecondaryFunctions() {
   }
 }
 
+Expression* ModuleSplitter::maybeLoadSecondary(Builder& builder,
+                                               Expression* callIndirect) {
+  if (config.jspi) {
```

Comment:
```suggestion
  if (!config.jspi) {
    return callIndirect;
```

It would be good to de-nest this function a bit.